lib/Rex/User/Linux.pm

```diff
@@ -359,6 +362,57 @@ sub get_user {
   );
 }
 
+sub lock_password {
+  my ( $self, $user ) = @_;
+
+  # Is the password already locked?
+  my $result = i_run "passwd --status $user";
+
+  die "Unexpected result from passwd: $result"
+    unless $result =~ /^$user\h+(L|NP|P)\h+/;
```

Comment:
On Sun, 2015-02-01 at 17:24 -0800, Ferenc Erki wrote:
> > +  die "Unexpected result from passwd: $result"
> > +    unless $result =~ /^$user\h+(L|NP|P)\h+/;
> 
> This breaks perl-5.8.9 compatibility, I'm going to change \h with
> normal space.

Maybe use \s instead? I just thought there might be times when it
returns more than one space or a tab (I haven't seen this, just thought
it might make the code less likely to break).